### DISCUSSION TITLE
Youtube plugin

### DISCUSSION THREAD

**Discussion by ghost** [Flow-Launcher/Flow.Launcher#1017] [Ideas]
I'm a huge fan of Flow Launcher, and i would like it to get a youtube extension, that could maybe search on youtube and display things like author, thumbnail etc...

**Comment by jjw24** [comment]
Hey how's it going, it's great that you are enjoying flow. I don't think so plugin exists yet, but yes would be very useful for those that YouTube alot.

**Comment by Garulf** [comment]
I could whip up a plugin fairly easily. You'd just have to provide a api key though.

**Comment by Garulf** [comment]
Okay I was able to create a plugin that uses YouTube's search function. As a bonus it doesn't need an API key to function!

https://github.com/Garulf/FlowYouTube/

![image](https://user-images.githubusercontent.com/535299/154381528-324fc4fd-db85-4209-83bd-218a593c214e.png)

**Reply by oSumAtrIX** [reply]
RE: https://github.com/Garulf/FlowYouTube/issues/1
